src/js/fc.js

```diff
@@ -27,6 +27,7 @@ var SERVO_RULES;
 var SERIAL_CONFIG;
 var SENSOR_DATA;
 var MOTOR_DATA;
+var MOTOR_OUTPUT_ORDER;
```

Comment:
Use `let`. It breaks the old style that are using the rest, but I think is better for the new code use the new standard.

Reply:
I add a message because I don't remember if I needed to use `window.MOTOR_OUTPUT_ORDER` or `let MOTOR_OUTPUT_ORDER` in other similar PR that I did. Check what works.